docs/intro.md

```diff
@@ -21,9 +21,12 @@ go get github.com/gofiber/fiber/v3
 ```
 
 ### Zero Allocation
-Some values returned from **fiber.Ctx** are **not** immutable by default.
+Because fiber is optimized for **high-performance** care is needed when using **fiber.Ctx**:
 
-Because fiber is optimized for **high-performance**, values returned from **fiber.Ctx** are **not** immutable by default and **will** be re-used across requests. As a rule of thumb, you **must** only use context values within the handler, and you **must not** keep any references. As soon as you return from the handler, any values you have obtained from the context will be re-used in future requests and will change below your feet. Here is an example:
+- **fiber.Ctx** is **not** thread-safe and **must not** be accessed concurrently by multiple goroutines, the immutable setting **does not** change this.
```

Comment:
The tone here is too aggresive. If `immutable` does not apply to some variables open a bug report so we can fix it.

Reply:
@omaskery Are you making a copy of the context object thougH?

Reply:
Could you clarify what you mean?

Reply:
@gaby well he wants to say that you should not make a copy of the ctx, because this object will be reused in the next process

he wanted to do this in his process, regardless of which methods of the ctx return copies of the values

@omaskery 
i agree with @gaby , i don't quite like the sound of the sentence, you should make this statement softer so as not to scare people off, because very few people want to copy the entire context and use it in passing

Reply:
Apologies, I'm struggling a little with the vagueness of the feedback regarding aggressiveness or tone. I'm not sure which parts of the text people are disagreeing factually with or finding tonally unpleasant.

Is there disagreement on whether `fiber.Ctx` is thread-safe? Is the issue that _some_ operations _are_ thread-safe and therefore the advice should not be as black-and-white as "no concurrent access"? Are there concerns that the immutable setting _should_ be guaranteeing that all operations on `fiber.Ctx` are thread-safe? Did I wrongly copy the use of bold emphasis from the surrounding text?

Reply:
@omaskery you can swap the order of the two points
then it would be okay for me